storyteller-api/cmd/api/main.go

```diff
@@ -15,10 +16,19 @@ import (
 func main() {
 	util.LoadEnv()
 
-	key := os.Getenv("CHAT_GPT_KEY")
+	//apiKey := os.Getenv("YANDEX_GPT_API_KEY")
+	apiGatewayID := os.Getenv("YANDEX_APIGATEWAY_ID")
+	catalogID := os.Getenv("YANDEX_CATALOG_ID")
 	prompt := os.Getenv("PROMPT")
 
-	var storyGenerator = storygenerator.NewOpenAIStoryGenerator(key, prompt)
+	//storyGenerator := storygenerator.NewYandexStoryGeneratorWithAPIKey(apiKey, catalogID, prompt)
+
+	iamTokenGenerator := iamgenerator.NewIAMTokenHTTP(apiGatewayID)
+	iamToken, err := iamTokenGenerator.GenerateToken()
+	if err != nil {
+		log.Printf("Can't get token %s", err)
```

Comment:
You must return from function after error

Reply:
fixed